smac_planner/include/smac_planner/node_se2.hpp

```diff
@@ -119,6 +122,27 @@ class NodeSE2
    */
   bool operator==(const NodeSE2 & rhs)
   {
+    // TODO(james-ward) Relaxing the tolerances on the goal match
```

Comment:
Should we remove this?

Reply:
Based on discussions about analytic solutions, yes.